tests/php/ORM/SQLSelectTest.php

```diff
@@ -1327,4 +1327,97 @@ public function testMultipleWithDuplicateName()
 
         $select->addWith('cte', new SQLSelect());
     }
+
+    public function subqueryProvider()
+    {
+        return [
+            'no-alias-string' => ['( SELECT DISTINCT "SQLSelectTest_DO"."ClassName" FROM "SQLSelectTest_DO") AS "FINAL"'],
```

Comment:
You've called this "no alias string" but the string does explicitly have an alias? It's a bit confusing, but not a deal breaker since it's just a test scenario identifier. I won't hold up the PR merging if this doesn't change.

Reply:
I meant to say that the alias wasn't explicitly defined in an array key.

I'll rename it to this.

```suggestion
            'no-explicit-alias-string' => ['( SELECT DISTINCT "SQLSelectTest_DO"."ClassName" FROM "SQLSelectTest_DO") AS "FINAL"'],
```

Reply:
Done